### DISCUSSION TITLE
Is it possible to see old documentation version

### DISCUSSION THREAD

**Discussion by oliviermattei** [primefaces/community#639] [PrimeVue]
Hello, 

I'm forced to freeze the version of primevue related to production constraints. I would know if it's possible to keep and get a custom primevue documentation version. In my case the 3.42. They are lots of update each version and I need to keep my 3.42 version

**Comment by tony-thomas-git** [comment]
Agreed. How do I access the documentation for my production version 3.39? For example, the page https://primevue.org/progressbar for the progress bar has new features and stackblitz that do not work with my version 3.39.

**Reply by j46r0** [reply]
I build it myself.

1. Go to [Releases](https://github.com/primefaces/primevue/releases).
2. Download desired version source.
3. Run `npm install`, `npm run build`, `npm run postinstall`, `npm run generate`.
4. And at last run `npm run preview` then go to [localhost](localhost:3000) in your browser.
